pygls/types.py

```diff
@@ -1236,6 +1251,15 @@ def __init__(self,
         self.context = context
 
 
+class HoverParams(TextDocumentPositionParams):
+    def __init__(self,
+                 textDocument: 'TextDocumentIdentifier',
+                 position: 'Position',
+                 workDoneToken: Optional[str] = None):
```

Comment:
and this parameter `work_done_token`

Reply:
With a look at pydantic this would propably need to be changed to make those types interoperate properly with that library. For now let's stick to snake_case, right.